velox/docs/functions/spark/binary.rst

```diff
@@ -7,29 +7,29 @@ Binary Functions
     Computes the hash of one or more input values using seed value of 42. For
     multiple arguments, their types can be different.
     Supported types are: BOOLEAN, TINYINT, SMALLINT, INTEGER, BIGINT, VARCHAR,
-    VARBINARY, REAL, DOUBLE, HUGEINT and TIMESTAMP.
+    VARBINARY, REAL, DOUBLE, HUGEINT, TIMESTAMP, ARRAY, MAP and ROW.
```

Comment:
HUGEINT is not a proper SQL type. Did you mean DECIMAL? Any particular reason not to say that all types are supported?

Reply:
Removed the "Supported types are: ..."

Reply:
@marin-ma  what's the Hugeint type? is it bigint( 8-byte signed integer numbers)? If so let's use bigint.